### ISSUE TITLE
When using ThreadStats() host tag is not sent with the metrics

### ISSUE THREAD

**Issue by harelba** [DataDog/datadogpy#180]
When using ThreadStats(), it seems that no host tag is being sent for metrics. 

I believe that if the user doesn't send (host=X) as a parameter to increment/gauge/etc., then the actual host name should be sent along with the metric, and not an empty host name (similar to the way that the datadog agent itself attaches the host tags to all metrics without the user having to know about it).

btw, when using ThreadStats.send_event() the host does get sent properly.

Thanks
Harel

**Comment by yannmh** [comment]
Thanks for opening.

I agree with you: to be consistent with the Datadog Agent, or Statsd metrics, we should attach the hostname to `ThreadStatsd` metrics unless specified not to.

We'll do the change for the 0.16 agent release 🚢 .

**Comment by harelba** [comment]
Thanks! that would be great.

For other people seeing this, I'll describe the simple workaround I used - I just reused your own method for getting the hostname and pushed it along with all my metrics as static tags. This makes sure that the data is sent using the same hostname, regardless of whether it's sent from within my app or from the datadog agent.

```
from datadog.util.hostname import get_hostname

datadog_discovered_hostname = get_hostname()
```
